consoles/node.html

```diff
@@ -20,7 +20,7 @@ <h1>Node</h1>
 {{ range query "up{job='node'}" | sortByLabel "instance" }}
 <tr>
   <td><a href="node-overview.html?instance={{ .Labels.instance }}">{{ reReplaceAll "(.*?://)([^:/]+?)(:\\d+)?/.*" "$2" .Labels.instance }}</a></td>
-  <td{{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
+  <td> {{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
```

Comment:
if you read what is after the `td` you will notice there is a condition and in each result of the condition you should see there is a `>` added each time.

So the code is correct from my point of view. 
What problems are you trying to solve here ?